packages/react/src/components/Accordion/__tests__/Accordion-test.js

```diff
@@ -5,6 +5,8 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import 'carbon-components/src/components/accordion/_accordion.scss';
```

Comment:
Would it make sense to also add an example test/assertion to fully validate the styles are included as we expect?

The `.#{$prefix}--accordion__content` initially has `display: none;` and could be asserted as such in the first "should render" test with `expect(getByText('Panel A')).not.toBeVisible();`

Reply:
I guess this would moreso apply to AccordionItem tests

Reply:
@tay1orjones I'm down either way, just let me know. When porting it over I was only looking at if the styles were being brought in or not

Reply:
That's fair, we'll validate this stuff as the refactor to RTL progresses

Reply:
How do we check if the styles are being brought in or not?

Reply:
@jnm2377 one quick way could be to change the `Accordion-test` to `console.log(document.styleSheets)` and see if one is loaded and explore the rules inside of it 👀  Another way could be to print out `document.head.innerHTML` which would print out the `<style>` tag that contains the styles